src/lib/profiles/security/WeaveCert.cpp

```diff
@@ -894,6 +901,509 @@ NL_DLL_EXPORT uint32_t SecondsSinceEpochToPackedCertTime(uint32_t secondsSinceEp
     return packedTime;
 }
 
+static WEAVE_ERROR GenerateCertSerialNumber(uint64_t & serialNum)
```

Comment:
Is most of the content below only needed by the weave tool? Does it successfully get dead-stripped out on devices?